File: .claude/skills/mycc/scripts/src/tasks/daily-report.py
#!/usr/bin/env python3
import subprocess, json

REPLY = '/home/vicki/.npm-global/lib/node_modules/@aster110/cc2wechat/dist/reply-cli.js'

def cmd(c):
    r = subprocess.run(c, shell=True, capture_output=True)
    return r.stdout.decode('utf8').strip()

def send(msg):
    subprocess.run(['node', REPLY, '--text', msg], capture_output=True)

try:
    nv = cmd('curl -s -o /dev/null -w "%{http_code}" https://integrate.api.nvidia.com/v1/models --max-time 5')
    prov = '✅ nvidia_nim' if nv == '200' else f'⚠️ nvidia_nim({nv})'
except:
    prov = '❌ nvidia_nim离线'

try:
    or_ = cmd('curl -s -o /dev/null -w "%{http_code}" https://openrouter.ai/api/v1/models --max-time 5')
    or_status = '✅ openrouter' if or_ == '200' else f'⚠️ openrouter({or_})'
except:
    or_status = '❌ openrouter'

disk = cmd("df -h / | tail -1 | awk '{print $5}'")
mem = cmd("free -h | awk '/Mem:/ {print $3\" / \"$2}'")
pm2 = json.loads(cmd('pm2 jlist'))
lines = []
for p in pm2:
    name = p['name']
    status = '✅在线' if p['pm2_env']['status']=='online' else '❌'+p['pm2_env']['status']
    lines.append(f"  {name:20s} {status}")
procs = '\n'.join(lines)

from datetime import datetime
now = datetime.now()
t = f"{now.hour:02d}:{now.minute:02d}"
msg = f"🌅 晨报 · {t}\n\n📊 系统状态\n{procs}\n\n💾 磁盘: {disk}\n🧠 内存: {mem}\n🌐 nvidia: {prov}\n🌐 openrouter: {or_status}\n\n祝今天顺利！"
send(msg)

File: .claude/skills/mycc/scripts/src/tasks/daily-report.ts
#!/usr/bin/env tsx
import { execSync } from 'child_process'
const REPLY = 'node /home/vicki/.npm-global/lib/node_modules/@aster110/cc2wechat/dist/reply-cli.js --text'
function cmd(c: string) { return execSync(c, { encoding: 'utf8' }).trim() }
function send(msg: string) { execSync(`${REPLY} "${msg.replace(/"/g, '\\"')}"`, { stdio: 'ignore' }) }
const pm2 = JSON.parse(cmd('pm2 jlist'))
const procs = pm2.map((p: any) => `  ${p.name.padEnd(20)} ${p.pm2_env.status==='online'?'✅在线':'❌'+p.pm2_env.status}`).join('\n')
const disk = cmd("df -h / | tail -1 | awk '{print $5}'")
const mem = cmd("free -h | awk '/Mem:/ {print $3\" / \"$2}'")
let prov='未知'
try{ const r=execSync('curl -s -o /dev/null -w "%{http_code}" https://integrate.api.nvidia.com/v1/models --max-time 5',{encoding:'utf8'}); prov=r==='200'?'✅ nvidia_nim':'⚠️ nvidia_nim('+r+')' }catch(){prov='❌ nvidia_nim离线'}
const now=new Date(); const t=`${String(now.getHours()).padStart(2,'0')}:${String(now.getMinutes()).padStart(2,'0')}`
send(`🌅 晨报 · ${t}\n\n📊 系统状态\n${procs}\n\n💾 磁盘: ${disk}\n🧠 内存: ${mem}\n🌐 Provider: ${prov}\n\n祝今天顺利！`)
